bin/ufcc

```diff
@@ -63,6 +63,91 @@ with warnings.catch_warnings(record=True) as w:
         print(mass_warn)
         warn2report.append(mass_warn)
 
+# For interactive selection of the groups for the contacts calculation
+if args.i_bool:
+    groups_dict = {
+        0: ['Database', target_system.database.selected],
+        1: ['Query', target_system.query.selected],
+        2: ['System', target_system.query.whole.universe.atoms],
+        3: ['Protein', target_system.query.whole],
+        4: ['Lipids', target_system.database.whole]
+    }
+
+    groups_to_add = target_system.database.whole.universe.select_atoms('not protein').groupby('resnames')
+    for label, ag in groups_to_add.items():
+        groups_dict[max(groups_dict.keys()) + 1] = ['{}'.format(label), ag]
+
+    def print_action_keys():
+        print('''\nActions keys:
+db  :  update database group for contacts calculation. (i.e. >> d Group_ID) where Group_ID is the identifier of the group on the list of groups.
+qr  :  update query group for contacts calculation. (i.e. >> d Group_ID) where Group_ID is the identifier of the group on the list of groups.
+gp :  create subgroups grouped by any topology attribute ("names", "resnames", "masses", etc). (i.e. >> gp Group_ID resnames) creates new groups for every different resname in the Group_ID selected.
+sl  :  create new groups by splitting a previous group at the desired level ("segment", "residue", "atom", "molecule"). (i.e. >> sl Group_ID residue) splits the Group_ID selected into all the residues that make it up, and create a new group for each of them.
+add:  merge two or more groups and create a new group with the combination. (i.e. >> add Group_ID1 Group_ID2 ... Group_IDn) creates a new group for the combination of the groups Group_ID1 Group_ID2 ... Group_IDn.
+del:  delete a group from the list of groups. (i.e. >> del Group_ID) remove the Group_ID selected.
+lg :  print the list of groups.
+h  :  print the help for all the available action keys.
+e  :  exit interactive selection and calculate de contacts between the Query and Database groups. 
```

Comment:
This is excellent! 

I can see a few ways we can improve this, however. One idea is to add support for `+` and `-`, which can be mapped into set operations (e.g. `+` can be `union`, `|` can be `intersection`, `-` can be `A - (A∩B)`, and others). 

This is not difficult to do, have a look at this: https://www.codingem.com/python-__add__-method/

But also note that from my understanding, MDAnalysis already implements many of these. I checked the code and it should be possible to add atomgroups directly (and do other operations as well): [link](https://github.com/MDAnalysis/mdanalysis/blob/b302e40ee45831409b5641311ac51fc4db08e07a/package/MDAnalysis/core/groups.py#L2394-L2448)